### DISCUSSION TITLE
Remove pinky column gap on 3x6 layout?

### DISCUSSION THREAD

**Discussion by juhakaup** [juhakaup/keyboards#27] [General]
I've been thinking about the ergonomics of the 3x6 layouts.

I don't use the 6 column version myself, but of course I have printed one for testing.
While I enjoy the quirkiness and ergonomics of pinky island on my 5 column keyboard, I can't say that I'm all that pleased with the amount of hand movement I have to do to reach that 6th column.

Now I wonder if I should rework it a bit by for example removing the gap between the 4th and the 5th columns.
This would bring the 6th column a bit closer, and therefore improve the ergonomics a bit.

If you use the 6 column version, I would like to hear your thoughts on this.

**Comment by senseored** [comment]
I use the 3x6 (both the unicolumn and non unicolumn) and on the unicolumn the gap is great - it sort of emulates the splay that I'm used to from choc boards. On the "standard/non unicolumn" the island feels a bit less great, but not bad per se.

I'd probably print out and try a 3x6 without the separation. I would also like to try moving the outermost column down maybe 0.25U, partly for visual interest but also I think it makes sense ergonomically
